l0_sampling/sketch.cpp

```diff
@@ -75,19 +116,26 @@ Sketch &operator+= (Sketch &sketch1, const Sketch &sketch2) {
 }
 
 bool operator== (const Sketch &sketch1, const Sketch &sketch2) {
-  return sketch1.n == sketch2.n && sketch1.seed == sketch2.seed &&
-    sketch1.num_bucket_factor == sketch2.num_bucket_factor &&
-    sketch1.bucket_a == sketch2.bucket_a &&
-    sketch1.bucket_c == sketch2.bucket_c &&
-    sketch1.already_quered == sketch2.already_quered;
+  if (sketch1.seed != sketch2.seed || sketch1.already_quered != sketch2.already_quered) 
+    return false;
+
+  for (size_t i = 0; i < Sketch::num_elems; ++i)
+  {
```

Comment:
nit(code style): open brace on prev line

Reply:
I'll change this